### PR TITLE
feat: add workflow name attribute

### PR DESCRIPTION
**Pull request by Marc-Roig**
### What does it do?

Adds new RW `name` attribute and adds a migration to set the default workflow name to "Default". 
This fixes a bug for the stage colors, they were updated on every startup.

### CODE REVIEW

File: packages/core/admin/ee/server/constants/default-workflow.json [hunk 1]
```diff
@@ -1 +1,3 @@
-{}
\ No newline at end of file
+{
+  "name": "Default"
```
Comment (gu-stav) [inline packages/core/admin/ee/server/constants/default-workflow.json:2]: Quick question: why is the default color not in here too?

File: packages/core/admin/ee/server/migrations/review-workflows-workflow-name.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+async function migrateReviewWorkflowName({ oldContentTypes, contentTypes }) {
+  // Look for RW name attribute
+  const hadName = !!oldContentTypes?.[WORKFLOW_MODEL_UID]?.attributes?.name;
+  const hasName = !!contentTypes?.[WORKFLOW_MODEL_UID]?.attributes?.name;
+
+  // Add the default workflow name if name attribute was added
+  if (!hadName && hasName) {
+    await strapi.query(WORKFLOW_MODEL_UID).updateMany({
+      data: {
+        name: 'Default',
```
Comment (gu-stav) [inline packages/core/admin/ee/server/migrations/review-workflows-workflow-name.js:14]: Should this use the constant json?

File: packages/core/admin/ee/server/constants/default-workflow.json (hunk 1 above)
Comment (Marc-Roig) [reply packages/core/admin/ee/server/constants/default-workflow.json:2]: workflow does not have a color no? 
For stages, https://github.com/strapi/strapi/blob/feature/review-workflow-name/packages/core/admin/ee/server/constants/default-stages.json I planned not to add the default color but to add different colors to each stage (based on a discussion with Maeva)

File: packages/core/admin/ee/server/migrations/review-workflows-workflow-name.js (hunk 2 above)
Comment (Marc-Roig) [reply packages/core/admin/ee/server/migrations/review-workflows-workflow-name.js:14]: I will use that :) nice one

**Review by jhoward1994 — APPROVED** [review]

**Pull request merged**